src/input-mask.test.ts

```diff
@@ -56,6 +56,19 @@ QUnit.test('Should mask telephone input and strip letters from entry', assert =>
   assert.equal(inputElement.value, '(123) 4  -    ');
 });
 
+QUnit.test('Should mask telephone input on blur', assert => {
+  InputMask.TelephoneInputMask(inputElement);
+  inputElement.dispatchEvent(TestUtilities.FocusEvent);
+  assert.equal(inputElement.value, '(   )    -    ');
+
+  inputElement.value = '(1234567890)   -    ';
+  inputElement.dispatchEvent(TestUtilities.BlurEvent);
+
+  window.requestAnimationFrame(() => {
+    assert.equal(inputElement.value, '(123)456-7890');
```

Comment:
@GarrettB71 this test should not pass. Unless my previous code is wrong, in which case we should fix that anyway. The value should be `(123) 456-7890`, note the space between the area code and the phone number.